Makefile

```diff
@@ -39,6 +39,8 @@ DOCKER_BUILD = docker build --no-cache \
 	    PLATFORM=$(PLATFORM) \
 	    BUILD_NUMBER=$(BUILD_NUMBER) \
 	    ENABLE_DHCP_GRAPH_SERVICE=$(ENABLE_DHCP_GRAPH_SERVICE) \
+	    PASSWORD=$(PASSWORD) \
```

Comment:
As Guohan mentioned below, there are default values in the rules/config, so the only way either value would be empty is if explicitly declared as such on the command line (e.g., `USERNAME=""`). This is a rare corner case and would only occur due to extreme user carelessness. I'm on the fence as to whether we need to check for it.